doc/man3/OSSL_PARAM_get_int32_t.pod

```diff
@@ -66,6 +70,14 @@ unsigned long int (ulong)
 
 =back
 
+OSSL_PARAM_TYPE() are a series of macros designed to assit initialising an
+array of OSSL_PARAM structures.
+Each of these macros defines a parameter of the specified B<TYPE> with the
+provided B<key> and paramter variable address B<addr>.
```

Comment:
parameter

Reply:
I think I should give up and go to sleep...